string nodeName
string errorMessage
string aditionalInfo
---
